DicomUltrasoundPlugin/DicomUltrasoundPlugin.py

```diff
@@ -701,6 +701,12 @@ def loadEigenArtemis3DUS(self,loadable):
     volumesLogic = slicer.modules.volumes.logic()
     outputVolume = volumesLogic.AddArchetypeScalarVolume(filePath,name,0,fileList)
 
+    ijk2ras = vtk.vtkMatrix4x4()
+    outputVolume.GetIJKToRASDirectionMatrix(ijk2ras)
```

Comment:
Are you sure you want to hardcode two values? If it is a LPS/RAS conversion then you should multiply the ijk2ras matrix by `diag(-1,-1,1,1)` matrix. If you want to keep this hardcoded setting of these two values then at least add a comment (why it is needed and why exactly those two values need to be changed like this).